### PR TITLE
Large secrets

### PR DESCRIPTION
**Pull request by paigerube14**
Using kube burner to create 10 secrets per namespace

https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-65206 

Based on Lena's script here: https://github.com/lenahorsley/svt/blob/9319303ce2becb9e76bf9c5aff4214893bdadfff/openshift_performance/ci/scripts/large_object_list/large_object_list.sh

https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/e2e-benchmarking-multibranch-pipeline/job/regression-test/403/console

### CODE REVIEW

File: perfscale_regression_ci/scripts/performance/large_secret_list.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+source ../common.sh
+source large_secret_list_env.sh
+
+echo "job iterations $JOB_ITERATION $PARAMETERS"
+echo "======Use kube-burner to load the cluster with test objects======"
+run_workload
+
+secret_count=$(oc get secrets -A -l test=listobject --no-headers | wc -l)
+expected_secrets=$(( $JOB_ITERATION * $SECRET_REPLICAS ))
+echo -e "\nTotal number of ${NAMESPACE} namespaces created: ${TOTAL_CLUSTERPROJECTS}"
```
Comment (qiliRedHat) [inline perfscale_regression_ci/scripts/performance/large_secret_list.sh:23]: @paigerube14 TOTAL_CLUSTERPROJECTS is not set anywhere. Did you intent to output the secret_count and expected_secrets here? That will be more meaningful than total projects to this case.

**Comment by lenahorsley** [comment]
@paigerube14 I'm okay with this. I ran the test and saw the expected and actual secrets in the log.
```
12-05 11:20:57.389  time="2023-12-05 16:20:57" level=info msg="Verifying created objects"
12-05 11:20:58.314  time="2023-12-05 16:20:58" level=info msg="secrets found: 1000 Expected: 1000"
12-05 11:20:58.314  time="2023-12-05 16:20:58" level=info msg="Job large-secrets took 50.17 seconds"
12-05 11:20:58.314  time="2023-12-05 16:20:58" level=info msg="Finished execution with UUID: 21638035-f785-4892-
```

File: perfscale_regression_ci/scripts/performance/large_secret_list.sh (hunk 1 above)
Comment (qiliRedHat) [reply perfscale_regression_ci/scripts/performance/large_secret_list.sh:23]: @paigerube14 Just a reminder if you missed this comment.

**Comment by qiliRedHat** [comment]
@paigerube14 I realized this PR isn't merged when I tried to run https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-65206. So I reopened it.

File: perfscale_regression_ci/scripts/performance/large_secret_list.sh (hunk 1 above)
Comment (qiliRedHat) [reply perfscale_regression_ci/scripts/performance/large_secret_list.sh:23]: @paigerube14 Please fix this and we can merge this PR.

**Pull request merged**